Gruntfile.circle.js

```diff
@@ -198,6 +199,18 @@ module.exports = function gruntConfig(grunt) {
 
     shell: {},
 
+    stylelint: {
+      options: {
+        configFile: `.stylelintrc`,
+        format: `css`
+      },
+      src: [
+        `./packages/**/*.css`,
+        `!./packages/*/node_modules/**/*.css`,
+        `!./packages/*/dist/**/*.css`
+      ]
+    },
+
```

Comment:
maybe change `./packages/**/*.css`, to `./packages/*/src/**/*.css`. I think the way you specified it, it walks the node modules tree, then excludes the node modules tree